page-objects/myProfilePage.ts

```diff
@@ -1,4 +1,4 @@
-import { Page, expect } from '@playwright/test';
+import { Page } from '@playwright/test';
 import { HelperBase } from './helperBase';
 
 
```

Comment:
Comments are related to all functions in this page:
1. function name should begin with lowercase letter
2. function name should be more descriptive (e.g. clickOnEditProfileButton, populateFirstName...), so that reader can be sure what is done in scope of function
3. instead of hardcoding values in the function, user FAKER library